src/test/run-pass/issue-30276.rs

```diff
@@ -1,14 +0,0 @@
-// Copyright 2013 The Rust Project Developers. See the COPYRIGHT
```

Comment:
Why are you removing this test?

Reply:
This is one of those things that don't trigger a typeck error but is totally broken, and I didn't want to deal with it.

Reply:
What does "totally broken" mean in this context?

Reply:
Something hid the ICE and #30276 was closed, but that was accidental. This is more or less a bug in WF-checking of function arguments in signatures, which @nikomatsakis somewhat insisted on keeping around, which allows `fn(Unsized)` to even exist (it shouldn't, for now).